### ISSUE TITLE
Disable DXGI capture in Screen Share

### ISSUE THREAD

**Issue by Khodyn** [powercord-community/suggestions#553]
### Description
Discord uses DXGI (full monitor capture) and crops when the streamed game is not occluded by any windows, otherwise it falls back to a different capture method.
### More info
This detection for if the window is occluded isn't very good. The detection can fail in these cases:
- Hold Alt+Tab and the window preview will show up in the stream.
- Minimize a window on top of the game and that window will be visible during the minimize animation.
- Because the detection is one frame early, focusing on the game can cause windows on top to appear for a frame.

tl;dr A plugin that forces Discord to never use DXGI capture, preventing leakage of other windows.

----

 - [x] I checked if this request wasn't already proposed
 - [x] I made sure this request follows Powercord's [plugin guidelines](https://powercord.dev/guidelines)
